worldedit-core/src/main/java/com/sk89q/worldedit/internal/cui/multi/MultiRegionClearEvent.java

```diff
@@ -0,0 +1,21 @@
+package com.sk89q.worldedit.internal.cui.multi;
+
+import com.sk89q.worldedit.internal.cui.CUIEvent;
+
+public class MultiRegionClearEvent implements CUIMultiEvent {
+    private final String regionID;
+
+    public MultiRegionClearEvent(String regionID) {
+        this.regionID = regionID;
+    }
+
+    @Override
+    public String getTypeId() {
+        return "+s";
```

Comment:
Is there a special meaning to `+s`? I would rather have something like `+clear`.

Reply:
On review of other files, it looks like a copy-paste mistake from `Shape`?

Reply:
the protocol was decided without our input :P "s" once meant selection. so sending "s|cuboid" meant "start a cuboid selection". the "+" part is for the multiregion protocol, so it doesn't clash with the normal WE selection CUI.